modules/configuration/pages/allow_and_deny_lists.adoc

```diff
@@ -0,0 +1,41 @@
+= Configure an Allow or Deny List
+:description: Learn how to configure an allow or deny list for a Redpanda Connect instance.
+
+From version 4.39.0, you can configure an allow or deny list to limit the Redpanda Connect components that users can run within data pipelines on a Redpanda Connect instance.
```

Comment:
It would be nice to keep the same ‘Introduced in’ sentence on a single for consistency. It helps users scan for the minimum required version.

Reply:
Updated